reflex/components/radix/themes/components.pyi

```diff
@@ -992,7 +992,7 @@ class TextField(TextFieldRoot, el.Input):
 
         Args:
             *children: The children of the component.
-            size: Text field size "1" - "3"
+            size: Specifies the visible width of a text control
```

Comment:
The inherited HTML element description probably shouldn't take precedent over the description from the class